yt/utilities/cosmology.py

```diff
@@ -388,10 +384,11 @@ def hubble_parameter(self, z):
         >>> print(co.hubble_parameter(1.0).in_units("km/s/Mpc"))
 
         """
-        return self.hubble_constant.in_base(self.unit_system) * self.expansion_factor(z)
+        return self.hubble_constant.in_base(self.unit_system) * \
+          self.expansion_factor(z)
```

Comment:
Minor nit, can you format this using parens to avoid the continuation character:

```
return (self.hubble_constant.in_base(self.unit_system) *
        self.expansion_factor(z))
```